index.html

```diff
@@ -13,17 +25,35 @@
 
 <body>
   <button onclick="topFunction()" id="myBtn" title="Go to top"><i class="fa fa-arrow-up"></i></button>
+
+  <div id="pulkit">
```

Comment:
dont use ur name as a id. Use 1 that describes what the id encloses

Reply:
i will change this one